src/ast/method.js

```diff
@@ -5,7 +5,7 @@
  */
 "use strict";
 
-const fn = require("./function");
+const Function = require("./function");
```

Comment:
Function is a global declaration - in some cases (don't remember well where) this kind of declaration can cause errors

Reply:
@ichiriac yes, you are right :+1: